### PR TITLE
added ability to shift-click abort Grand Exchange offers.

### LINKED ISSUE #9102: GE Abort Shift Click
**Is your feature request related to a problem? Please describe.**
When you're clearing your bank or selling a high quantity of items quite a few may not sell even with 5% lowered which obviously becomes a little tedious and this would be just for Quality of Life.

**Describe the solution you'd like**
What I was thinking was there was a right click option for you to abort the item either for buying or selling. I would believe it would fit under Shift Click Menu.
Screenshot_28

### PR DESCRIPTION
**Pull request by Adelaidian**
Added Shift Click to Abort Offers in the Grand Exchange.

Closes #9102

### CODE REVIEW

**Comment by xDemoN** [comment]
Isn't this the same as your original PR, #9127 ?

**Comment by Hydrox6** [comment]
@xDemoN yes, but this one is in the correct location, from the feedback on that PR

File: runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java [hunk 1]
```diff
@@ -162,6 +162,16 @@ default FairyRingMode swapFairyRing()
 		return FairyRingMode.LAST_DESTINATION;
 	}
 
+	@ConfigItem(
+		keyName = "swampGrandExchangeAbort",
```
Comment (JohnDonavon) [inline runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: typo: "swampGrandExchangeAbort" -> "swapGrandExchangeAbort"

Comment (Adelaidian) [reply runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java]: 😆 Thank you, must be watching too much Swampletics

**Comment by Adam-** [comment]
added in 61a72d5a54798b606e12bb76deaa9ef2c12fb981

**Pull request closed without merging**